### PR TITLE
feat: Added protocol strandedness inference after mapping with STAR, …

### PR DESCRIPTION
**Pull request by ericblanc20**
…preparation for cbioportal_export

### CODE REVIEW

File: snappy_wrappers/wrappers/rseqc/wrapper.py [hunk 1]
```diff
@@ -17,63 +26,114 @@
...
-    then
-        strand=2
-    fi
+# ----- Infer protocol strandedness
+infer=0
+if [ $(echo "$forward > {config[threshold]}" | bc -l) -gt 0 ]
+then
+    infer=1
+fi
+if [ $(echo "$reverse > {config[threshold]}" | bc -l) -gt 0 ]
```
Comment (mbenary) [inline snappy_wrappers/wrappers/rseqc/wrapper.py:82]: which strand is taken if both strand counts are above the threshold? seems to be the reverse always.

File: snappy_pipeline/workflows/ngs_mapping/__init__.py [hunk 2]
```diff
@@ -410,6 +410,10 @@
       trim_adapters: false
       mask_duplicates: false
       include_unmapped: true
+    strandedness:
+      path_exon_bed: REQUIRED
+      strand: -1
```
Comment (mbenary) [inline snappy_pipeline/workflows/ngs_mapping/__init__.py:415]: Add comment/explanation for other options for easier understanding.

File: snappy_wrappers/wrappers/rseqc/wrapper.py (hunk 1 above)
Comment (ericblanc20) [reply snappy_wrappers/wrappers/rseqc/wrapper.py:82]: The threshold represents the proportion of reads that map in the forward or reverse direction. It must be greater than 0.5. Usually, it should be set around 0.8-0.9. If neither forward nor reverse reach this value, the protocol is assumed to be unstranded (0).

File: snappy_pipeline/workflows/ngs_mapping/__init__.py [hunk 3]
```diff
@@ -410,6 +410,10 @@
...
+      threshold: 0.1
```
[the remark this answers is not in the document]
Comment (ericblanc20) [reply snappy_pipeline/workflows/ngs_mapping/__init__.py:416]: Correct!

**Review by mbenary — APPROVED** [review]

**Pull request merged**